js-miniapp-sdk/README.md

```diff
@@ -70,8 +70,9 @@ Note that there are two types of permissions:
 
 | Permission | Type | Method | Description |
 | --- | --- | --- | --- |
-| LOCATION | Device | `requestLocationPermission()` | The current location of the device.<br>Data position can be accessible via geolocation request (`navigator.geolocation`). |
-| User Data | Custom | `requestCustomPermission()` | Custom permissions for User Data. Currently supported types are `CustomPermissionName.USER_NAME`, `CustomPermissionName.PROFILE_PHOTO`, and `CustomPermissionName.CONTACT_LIST`. These permisions should be requested before you attempt to access the User's data. |
+| Location | Device | `requestLocationPermission()` | The current location permission of the device.<br>Data position can be accessible via geolocation request (`navigator.geolocation`). |
+| Location | Custom | `requestCustomPermissions()` | Custom permission for device location. Currently supported type is `CustomPermissionName.LOCATION`. |
```

Comment:
I think you can remove this line and add `Custom permission for device location. Currently supported type is CustomPermissionName.LOCATION` to the section above. Also, we need to update the `Device permissions` content [here](https://github.com/rakutentech/js-miniapp/blob/3d1279368c2fe735033f783ad5e5af34aa944762/js-miniapp-sdk/README.md#3-request-permissions) to explain that device permission is also custom permission now, it is required custom permission to be processed first.

Reply:
I have updated this part. 

> Also, we need to update the Device permissions content here to explain that device permission is also custom permission now.

Ah, I think, not all device permissions require custom permissions except location at the moment.